doc/classes/RenderSceneData.xml

```diff
@@ -51,5 +51,41 @@
 				[b]Note:[/b] If a single view is rendered, this returns the camera projection. If more than one view is rendered, this will return a projection for the given view including the eye offset.
 			</description>
 		</method>
+    <method name="decal_atlas_get_texture" qualifiers="const">
+      <return type="RID" />
+      <description>
+        Returns the [RID] of the decal atlas texture.
+      </description>
+    </method>
+    <method name="decal_atlas_get_texture_srgb" qualifiers="const">
+      <return type="RID" />
+      <description>
+        Returns the [RID] of the decal atlas texture in srgb.
+      </description>
+    </method>
+    <method name="directional_shadow_get_texture" qualifiers="const">
+      <return type="RID" />
+      <description>
+        Returns the [RID] of the directional shadow atlas texture.
+      </description>
+    </method>
+    <method name="get_directional_light_buffer" qualifiers="const">
+      <return type="RID" />
+      <description>
+        Return the [RID] of the directional light buffer containing the scene directional light data per light source as a UBO.
```

Comment:
I think this is right?
```suggestion
        Returns the [RID] of the directional light buffer, which contains the scene directional light data for each light source as a UBO.
```
Could be written similarly also for `get_omni_light_buffer` and `get_spot_light_buffer`